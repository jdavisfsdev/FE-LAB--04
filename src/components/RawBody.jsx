/* eslint-disable max-len */
import React from 'react';
import RawBodyCSS from './RawBody.module.css';

export default function RawBody({ onChange, body }) {
  return (
    <div>
      <input className={RawBodyCSS.input} onChange={onChange} name="body" placeholder="Raw JSON" type="text" value={body} />
    </div>
  );
}
