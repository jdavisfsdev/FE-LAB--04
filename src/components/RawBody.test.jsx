import React from 'react';
import { render } from '@testing-library/react';
import RawBody from './RawBody';

describe('RawBody componenet snapshot', () => {
  it('Renders the raw body input box', () => {
    const { asFragment } = render(
      <RawBody />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
