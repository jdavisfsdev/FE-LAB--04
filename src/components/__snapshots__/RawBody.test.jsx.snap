// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RawBody componenet snapshot Renders the raw body input box 1`] = `
<DocumentFragment>
  <div>
    <input
      class="input"
      name="body"
      placeholder="Raw JSON"
      type="text"
      value=""
    />
  </div>
</DocumentFragment>
`;
